nest/nest.py

```diff
@@ -1526,28 +1526,38 @@ def eta_begin(self):
         if 'eta_begin' in self._structure:
             return parse_time(self._structure['eta_begin'])
 
+    def _set_eta(self, trip_id, eta_begin, eta_end):
+        if self.num_thermostats == 0:
+            raise ValueError("ETA can only be set or cancelled when a"
+                             " thermostat is in the structure.")
+        if trip_id is None:
+            raise ValueError("trip_id must be not None")
+
+        data = {'trip_id': trip_id,
+                'estimated_arrival_window_begin': eta_begin,
+                'estimated_arrival_window_end': eta_end}
```

Comment:
```python
data = {'trip_id': trip_id,
            'estimated_arrival_window_begin': eta_begin}
if eta_end is not None:
    data[estimated_arrival_window_end] = eta_end
```

Reply:
I confirmed that `estimated_arrival_window_end` must be included even if `estimated_arrival_window_begin` is set to `int(0)`